services/harmony/env-defaults

```diff
@@ -225,7 +225,7 @@ LOG_STDOUT=false
 # a variable name of GIOVANNI_ADAPTER_IMAGE you would specify giovanni-adapter (converting to lowercase
 # and dash case and dropping _IMAGE. Make sure if the image for the service is not publicly available
 # that you have built the docker image locally, otherwise the service will fail to start.
-LOCALLY_DEPLOYED_SERVICES=harmony-service-example
+LOCALLY_DEPLOYED_SERVICES=harmony-service-example,opera-rtc-s1-browse,hybig
```

Comment:
We'll revert the change to this line before we merge into nasa/harmony

Reply:
The Harmony team points out that we can override any of these settings in our local `.env` file to facilitate local development, so we can avoid committing one-off changes like this in the future.